docs/explanation/api-and-clients.md

```diff
@@ -18,13 +18,24 @@ In addition to the Go client, there's also a [Python client](https://github.com/
 
 API endpoints fall into one of three access levels, from least restricted to most restricted:
 
-* `untrusted`: these are allowed from any user, even unauthenticated users using the HTTP-over-TCP listener. The only untrusted endpoints are `/v1/system-info` and `/v1/health`.
-* `read`: these are allowed from any authenticated user, regardless of access level. They are usually read operations that use the HTTP `GET` method, such as listing services or viewing notices.
-* `admin`: these are only allowed from admin users. They are usually write or modify operations that use the HTTP `POST` method, such as adding a layer or starting a service.
+* **Open-access** - Allowed from any user, even unauthenticated users using the HTTP-over-TCP listener.
+    * `GET /v1/system-info`, which returns the Pebble version and an identifier for the daemon
```

Comment:
It's actually an identifier for this specific boot. Could just say "Pebble version and other information", and leave details to Tiexin's API page.

Reply:
Changed - thanks!